airbyte-cdk/bulk/core/extract/src/test/kotlin/io/airbyte/cdk/read/StateManagerStreamStatesTest.kt

```diff
@@ -185,10 +185,6 @@ class StateManagerStreamStatesTest {
         val eventsStream: Stream = stateManager.feeds.mapNotNull { it as? Stream }.first()
         Assertions.assertEquals("EVENTS", eventsStream.name)
         Assertions.assertEquals(listOf("MSG", "ID", "TS"), eventsStream.fields.map { it.id })
-        Assertions.assertEquals(
-            listOf(listOf("ID")),
-            eventsStream.primaryKeyCandidates.map { col -> col.map { it.id } },
-        )
```

Comment:
Why was this test removed? It is correct for a single PK `source_defined_primary_key=[[ID]]`

Reply:
We no longer have the `primaryKeyCandidates` field in the `Stream`, as `configuredPrimaryKey` makes it redundant